guides/common/modules/proc_configuring-project-to-use-ldap.adoc

```diff
@@ -16,26 +17,24 @@ For more information, see:
 ----
 # setsebool -P nis_enabled on
```

Comment:
Where do I run this command? On the Project Server?

Reply:
Yes, Satellite is the machine that initiates outgoing LDAP query.